cpp/collections_module/algorithms/pairs.hpp

```diff
@@ -0,0 +1,36 @@
+#include <vector>
+
+#include <mgp.hpp>
+
+constexpr std::string_view kResultPairs = "pairs";
+
+void Pairs(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory) {
+  mgp::memory = memory;
+  const auto arguments = mgp::List(args);
+  const auto record_factory = mgp::RecordFactory(result);
+  try {
+    mgp::List pairsList = mgp::List();
+
+    const auto &inputList = arguments[0].ValueList();
+
+    if (!inputList.Size() == 0) {
```

Comment:
You can check here if `inputList` is empty and just return. This concept is called early return. https://medium.com/swlh/return-early-pattern-3d18a41bba8. It looks like this. and after you continue doing regular checks. This way you reduce indentation for one.